.gitignore

```diff
@@ -1,3 +1,6 @@
 /node_modules
 .DS_Store
 /package-lock.json
+index.html
+index.js
+/docs
```

Comment:
The /docs folder needs to remain in the repository as it contains the project's demo code and reference documentation. It's also the source that GithHub Pages uses to host the demo and API docs for the project.